### ISSUE TITLE
Language

### ISSUE THREAD

**Issue by magicse** [k2-fsa/sherpa-onnx#454]
Hello. Thank you for Your work.
https://colab.research.google.com/drive/1cI9VzlimS51uAw4uCR-OBeSXRPBc4KoK?usp=sharing
I try convert vit model to onnx for my languagem, and all test good with onnxruntime.
But in  step with test with sherpa-onnx it say that my language unknown by sherpa.
Could you  please add Ukrainian language?
[Model](https://github.com/magicse/Ukr-Sherpa-ONNX-model/releases/download/1.0/Vit_TTS_ONNX.zip)
```
%%shell
sherpa-onnx-offline-tts \
  --vits-model=./model.onnx \
  --vits-lexicon=./lexicon.txt \
  --vits-tokens=./tokens.txt \
  --output-filename=./test.wav \
  "Вітаю, як ваші справи? Що ви тут робите?"
  
  /project/sherpa-onnx/csrc/parse-options.cc:Read:361 sherpa-onnx-offline-tts --vits-model=./model.onnx --vits-lexicon=./lexicon.txt --vits-tokens=./tokens.txt --output-filename=./test.wav 'Вітаю, як ващі справи?. Що ви тут робите?' 

/project/sherpa-onnx/csrc/lexicon.cc:InitLanguage:321 Unknown language: Ukrainian
---------------------------------------------------------------------------
CalledProcessError                        Traceback (most recent call last)
[<ipython-input-50-490d59815704>](https://localhost:8080/#) in <cell line: 1>()
----> 1 get_ipython().run_cell_magic('shell', '', 'sherpa-onnx-offline-tts \\\n  --vits-model=./model.onnx \\\n  --vits-lexicon=./lexicon.txt \\\n  --vits-tokens=./tokens.txt \\\n  --output-filename=./test.wav \\\n  "Вітаю, як ваші справи? Що ви тут робите?"\n')

3 frames
[/usr/local/lib/python3.10/dist-packages/google/colab/_system_commands.py](https://localhost:8080/#) in check_returncode(self)
    135   def check_returncode(self):
    136     if self.returncode:
--> 137       raise subprocess.CalledProcessError(
    138           returncode=self.returncode, cmd=self.args, output=self.output
    139       )

CalledProcessError: Command 'sherpa-onnx-offline-tts \
  --vits-model=./model.onnx \
  --vits-lexicon=./lexicon.txt \
  --vits-tokens=./tokens.txt \
  --output-filename=./test.wav \
  "Вітаю, як ващі справи?. Що ви тут робите?"
' returned non-zero exit status 255.
```

**Comment by csukuangfj** [comment]
please wait for #453

**Comment by csukuangfj** [comment]
![Screenshot 2023-11-29 at 18 52 26](https://github.com/k2-fsa/sherpa-onnx/assets/5284924/1bf4f574-7bb4-494b-b21b-c000d8d39275)



@magicse 

Please try Ukrainian tts models using our huggingface space.

https://huggingface.co/spaces/k2-fsa/text-to-speech

**Comment by magicse** [comment]
I tested and it work ok.
Wave duration : 1.760 s
Processing time: 0.072 s
RTF: 0.072/1.760 = 0.041

Could you try convert this model instead lada?
https://github.com/coqui-ai/TTS/releases/download/v0.8.0_models/tts_models--uk--mai--vits.zip
Because it has more pleasure voice.

**Comment by magicse** [comment]
Hi @csukuangfj 
could you please add to  https://huggingface.co/spaces/k2-fsa/text-to-speech
this model https://github.com/coqui-ai/TTS/releases/download/v0.8.0_models/tts_models--uk--mai--vits.zip ?

**Comment by csukuangfj** [comment]
> Hi @csukuangfj could you please add to https://huggingface.co/spaces/k2-fsa/text-to-speech this model https://github.com/coqui-ai/TTS/releases/download/v0.8.0_models/tts_models--uk--mai--vits.zip ?

@magicse 

I am happy to tell you that I just managed to convert it. Please wait for a moment before I add it to sherpa-onnx.

**Comment by csukuangfj** [comment]
Fixed by #469
